rpm/treat_all_32bit_arms_as_eabi.patch

```diff
@@ -2,30 +2,30 @@ diff --git a/lib/config.guess b/lib/config.guess
 index 256083a70..40fdd5e96 100755
 --- a/lib/config.guess
 +++ b/lib/config.guess
-@@ -2,7 +2,7 @@
- # Attempt to guess a canonical system name.
- #   Copyright 1992-2018 Free Software Foundation, Inc.
+@@ -4,7 +4,7 @@
```

Comment:
Do we still need this patch? The workaround shouldn't be necessary anymore?

Reply:
Seems to work without this patch. Comment in the patch mentions it needs rpm update which has been done long after this patch was added so should be fine.